tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_section_order.py

```diff
@@ -0,0 +1,220 @@
+        case (SectionKind.TYPE, SectionTest.WRONG_ORDER, CasePosition.HEADER):
+            return (
+                "ef000102000100030100040400010000800001305000ef",
+                EOFException.MISSING_TYPE_HEADER,
+            )
+        case (SectionKind.TYPE, SectionTest.WRONG_ORDER, CasePosition.BODY):
+            return (
+                "ef000101000402000100030400010030500000800001ef",
+                # TODO why invalid first section type? it should say that the body incorrect
+                EOFException.INVALID_FIRST_SECTION_TYPE,
+            )
+        case (SectionKind.TYPE, SectionTest.WRONG_ORDER, CasePosition.BODY_AND_HEADER):
+            return (
+                "ef000102000100030100040400010030500000800001ef",
+                EOFException.MISSING_TYPE_HEADER,
+            )
+        case (SectionKind.CODE, SectionTest.MISSING, CasePosition.HEADER):
+            return "ef00010100040400010000800001305000ef", EOFException.MISSING_CODE_HEADER
+        case (SectionKind.CODE, SectionTest.MISSING, CasePosition.BODY):
+            return (
+                "ef000101000402000100030400010000800001ef",
+                # TODO should be an exception of empty code bytes, because it can understand that
+                # last byte is data section byte
+                EOFException.INVALID_SECTION_BODIES_SIZE,
+            )
+        case (SectionKind.CODE, SectionTest.MISSING, CasePosition.BODY_AND_HEADER):
+            return "ef00010100040400010000800001ef", EOFException.MISSING_CODE_HEADER
+        case (SectionKind.CODE, SectionTest.WRONG_ORDER, CasePosition.HEADER):
+            return (
+                "ef000101000404000102000100030000800001305000ef",
+                EOFException.MISSING_CODE_HEADER,
+            )
+        case (SectionKind.CODE, SectionTest.WRONG_ORDER, CasePosition.BODY):
+            return (
+                "ef000101000402000100030400010000800001ef305000",
+                EOFException.UNDEFINED_INSTRUCTION,
+            )
+        case (SectionKind.CODE, SectionTest.WRONG_ORDER, CasePosition.BODY_AND_HEADER):
+            return (
+                "ef000101000404000102000100030000800001ef305000",
+                EOFException.MISSING_CODE_HEADER,
+            )
+        case (SectionKind.DATA, SectionTest.MISSING, CasePosition.HEADER):
+            return "ef000101000402000100030000800001305000ef", EOFException.MISSING_DATA_SECTION
+        case (SectionKind.DATA, SectionTest.MISSING, CasePosition.BODY):
+            return (
+                "ef000101000402000100030400010000800001305000",
+                # TODO we declared that data has 1 byte, but didn't provide any bytes
+                None,
+            )
+        case (SectionKind.DATA, SectionTest.MISSING, CasePosition.BODY_AND_HEADER):
+            return "ef000101000402000100030000800001305000", EOFException.MISSING_DATA_SECTION
+        case (SectionKind.DATA, SectionTest.WRONG_ORDER, CasePosition.HEADER):
```

Comment:
hm thats because I don't touch type and thus have only `code` and `data` to switch places. 
can resolve if I add subcontainer, but this test does not touch subcontainer logic